client/lib/feed-stream-store/index.js

```diff
@@ -202,6 +202,7 @@ function getStoreForRecommendedPosts( storeId ) {
 				break;
 			case 'custom_recs_posts_with_images':
 				query.algorithm = 'read:recommendations:posts/es/7';
+				query.seed = random( 0, 10000 );
```

Comment:
it was the product of a discussion between @aaronyan and I.
We needed to pick a range large enough that we didn't think someone would get the same one twice too frequently but also no reason to go overboard and chance an error in transmission (between js-->php-->es).

You make a nice case in point :).  We don't want to a new seed on each request BUT a new seed on each page refresh. I should add a comment